vip-integrations.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * Plugin Name: VIP Integrations
+ * Description: Plugin loading for VIP plugin integrations.
+ * Author: Automattic
+ * License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace Automattic\VIP\Integrations;
+
+defined( 'ABSPATH' ) || die();
+
+require_once __DIR__ . '/vip-integration-helper/integrations/integrations.php';
+require_once __DIR__ . '/vip-integration-helper/integrations/block-data-api.php';
+
+add_action( 'muplugins_loaded', __NAMESPACE__ . '\\register_integrations', /* priority */ 4 );
+
+/**
+ * Register valid integrations.
+ */
+function register_integrations(): void {
```

Comment:
```suggestion
function register_vip_integrations(): void {
```

Just helps to differentiate it a little from `Integrations::register()`.